qfieldsync/gui/cloud_transfer_dialog.py

```diff
@@ -323,7 +357,38 @@ def prepare_project_transfer(self):
             self.openProjectCheck.setVisible(False)
             return
 
-        if len(list(self.cloud_project.files_to_sync)) == 0:
+        # Remove files that are already present remotely, override should be done manually
+        if self.localized_datasets_project and self.localized_datasets_files:
+            localized_data_paths = (
+                QgsApplication.instance().localizedDataPathRegistry().paths()
+            )
+            localized_datasets_project_files = (
+                self.localized_datasets_project.get_files()
+            )
+            localized_datasets_project_names = []
```

Comment:
```suggestion
            localized_datasets_project_filenames = []
```
I guess those are filenames, no?

Reply:
@suricactus , no, they are ProjectFile.name (i.e. the relative name string to the root of the local dir)